sdcm/sct_config.py

```diff
@@ -1310,7 +1310,11 @@ class SCTConfiguration(dict):
              help="Number of of raid level: 0 - RAID0, 5 - RAID5"),
 
         dict(name="bare_loaders", env="SCT_BARE_LOADERS", type=boolean,
-             help="Don't install anything but collectd to the loaders during cluster setup")
+             help="Don't install anything but collectd to the loaders during cluster setup"),
+
+        dict(name="disable_argus", env="SCT_DISABLE_ARGUS", type=boolean,
```

Comment:
I would really prefer to have identical meanings of the name and behavior.

Examples:
```
No:  disable_argus=True
Yes: enable_argus=False
```

Reply:
I am not sure what you mean. But if you mean that we should have enable_argus instead of disable_argus parameter,
you should be aware that it is going to change the default behavior of the system since, not having this parameter in the
configuration is going to disable argus by default. If you think it is preferable I can make the change.

Reply:
We have `defaults/test_default.yaml` file where we init all defaults.
So, you can just place there "enabled_argus=true" and it will be "existing behavior".

Reply:
I changed it to enable_argus